### PR TITLE
Updating Gallery Files

### PR DESCRIPTION
**Pull request by phantasmino**
This is Ebbi's submission, please review. I will add more. Thanks.

### CODE REVIEW

**Comment by phantasmino** [comment]
I'm unfamiliar with the syntax, just let me know if this is good. Also, should I merge instead into ika's branch? thanks.

File: content/gallery/RyoD - Oasis of Sant'Alessio.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: RyoD - Oasis of Sant'Alessio
+coords: [45.21972, 28.15361]
+images:
+    - https://media.discordapp.net/attachments/1119383864763232266/1131487953458319450/photo_5886590663485667934_y.jpg?width=772&height=579
+	- https://media.discordapp.net/attachments/1119383864763232266/1131487953730945084/photo_5886590663485667935_y.jpg?width=433&height=577
```
Comment (ikanexus) [inline content/gallery/RyoD - Oasis of Sant'Alessio.md:6]: could you change the tabs on lines 6-13  to spaces like in line 5 please? yaml doesn't like tabs  :(

**Comment by ikanexus** [comment]
would you be able to rename the files as well to be more 'snake-case'. e.g. `RyoD - Oasis of Sant'Alessio.md` -> `ryod-oasis-of-sant-alessio.md`?  I'm not sure how well nuxt-content works with spaces and special charaters

**Comment by phantasmino** [comment]
Got it Ika. Let me know if these changes work. Are the special characters inside the markdown file on the name field okay?

**Comment by Tougrel** [comment]
You should follow Ika's format example and not add `_` on places where a space would have been present (except in the place's name) so we don't have different formats on the files. Current format for the files is `{name}-{place}.md` so with that in mind the following files should be changes to:

- `RyoD_-_Oasis_Sant-Alessio` -> `ryod-oasis_sant_alessio`
- `phantasma_-_Bosquet_de_Lencelade_-_Versailles` -> `phantasma-bosquet_de_lencelade_versailles`
- `phantasma_-_Edinburgh_Graveyard` -> `phantasma-edinburgh_graveyard`

Also the name inside the markdown files shouldn't include the place. We can add another variable for that later on if we need it 👍

**Review by ikanexus — APPROVED** [review]

**Comment by ikanexus** [comment]
@phantasmino Can you request a review when you're ready - not sure if you want to add more or do more PRs if you want to add more

**Comment by phantasmino** [comment]
Sure, I'm going to temporarily remove Ebbi's update from my commit and then I'll request review.

**Review by ikanexus — APPROVED** [review]

**Pull request merged**